### PR TITLE
[jsk_naoqi_robot] add comment on ROS_DISTRO

### PR DESCRIPTION
**Pull request by kochigami**
If there is any problem, please let me know.

### CODE REVIEW

File: jsk_naoqi_robot/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ cd  catkin_ws
 wstool init src
 wstool merge -t src https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_naoqi_robot/pepper.rosinstall
 wstool update -t src
+;; Be sure that ROS_DISTRO is set (echo $ROS_DISTRO). If not, execute "source /opt/ros/indigo/setup.bash"
 rosdep install -y -r --from-paths src --ignore-src
```
Comment (furushchev) [inline jsk_naoqi_robot/README.md]: @kochigami updating this line to
` rosdep install -y -r --from-paths src --ignore-src --rosdistro=XXXX`
is better than just documenting.
Also `;;` is for lisp not a shell.

Comment (k-okada) [inline jsk_naoqi_robot/README.md]:
```
;; Be sure
```
should be
```
# Be sure
```

**Comment by kochigami** [comment]
Thank you very much for your comments.
I updated information on 
- ros install
- ROS_DISTRO
- install python naoqi sdk 
- roslaunch jsk_pepper_startup.launch  

They are what I got stuck when I set up Nao/ Pepper ros environment with new PC.
If it has still any problem, please let me know.

**Comment by k-okada** [comment]
I think
```
source /opt/ros/indigo/setup.bash
rosdep install -y -r --from-paths src --ignore-src
```
is better, because when you run `catkin build` we need to source setup.bash

```
k-okada@p40-yoga:/tmp/hoge$ unset ROS_DISTRO; unset ROS_PACKAGE_PATH; unset
ROS_ROOT; unset CMAKE_PREFIX_PATH
k-okada@p40-yoga:/tmp/hoge$ catkin b
==> Expanding alias 'b' from 'catkin b' to 'catkin build'
----------------------------------------------
Profile:                     default
Extending:                   None
Workspace:                   /tmp/hoge
----------------------------------------------
Source Space:       [exists] /tmp/hoge/src
Log Space:          [exists] /tmp/hoge/logs
Build Space:        [exists] /tmp/hoge/build
Devel Space:        [exists] /tmp/hoge/devel
Install Space:      [unused] /tmp/hoge/install
DESTDIR:            [unused] None
----------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        None
----------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
----------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
----------------------------------------------


----------------------------------------------
WARNING: Your workspace is not extending any
other result space, but it is set to use a
`linked` devel space layout. This requires
the `catkin` CMake package in your source
space in order to be built.
----------------------------------------------

[build] No packages were found in the source space '/tmp/hoge/src'
[build] No packages to be built.
[build] Package table is up to date.

[build] Summary: All 0 packages succeeded!

[build]   Ignored:   None.

[build]   Warnings:  None.

[build]   Abandoned: None.

[build]   Failed:    None.

[build] Runtime: 0.2 seconds total.

k-okada@p40-yoga:/tmp/hoge$ source /opt/ros/indigo/setup.bash
k-okada@p40-yoga:/tmp/hoge$ catkin b
==> Expanding alias 'b' from 'catkin b' to 'catkin build'
----------------------------------------------
Profile:                     default
Extending:             [env] /opt/ros/indigo
Workspace:                   /tmp/hoge
----------------------------------------------
Source Space:       [exists] /tmp/hoge/src
Log Space:          [exists] /tmp/hoge/logs
Build Space:        [exists] /tmp/hoge/build
Devel Space:        [exists] /tmp/hoge/devel
Install Space:      [unused] /tmp/hoge/install
DESTDIR:            [unused] None
----------------------------------------------
Devel Space Layout:          linked
Install Space Layout:        None
----------------------------------------------
Additional CMake Args:       None
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
Cache Job Environments:      False
----------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
----------------------------------------------
Workspace configuration appears valid.
----------------------------------------------
[build] No packages were found in the source space '/tmp/hoge/src'
[build] No packages to be built.
[build] Package table is up to date.

[build] Summary: All 0 packages succeeded!

[build]   Ignored:   None.

[build]   Warnings:  None.

[build]   Abandoned: None.

[build]   Failed:    None.

[build] Runtime: 0.2 seconds total.
```

--
◉ Kei Okada


2016-10-14 14:35 GMT+09:00 Kanae Kochigami <notifications@github.com>:

**Comment by kochigami** [comment]
Thank you very much for your comment. 
I updated the pull request as you mentioned.

**Comment by k-okada** [comment]
thank you

**Pull request merged**